WebApp/autoreduce_webapp/templates/admin/graph_home.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Graphing Home</title>
+    <style>
+        body {
+            font-size: 20px;
+        }
+        #list-container {
+            width: 500px;
+        }
+        li {
+            margin-top: 20px;
+        }
+
+        input[type~="number"] {
+            width: 50px;
+            margin-left: 10px;
+        }
+    </style>
+</head>
+<body>
+    <h3>Insturments</h3>
+    <p>Choose the number of runs you want to display on the graph and click 'Graph'.</p>
+    <div id="list-container">
+        <ul>
+        {% for instrument in instruments %}
+            <li><form action="/graph/{{ instrument.name }}">{{ instrument.name }}<input type="number" name="last" min="1" max="2000" value="50"> most recent runs <input value="Graph" type="submit"> </form></li>
```

Comment:
Currently if you leave the box blank and then submit you get a `VALUE ERROR` page. 
If we can fix this in the prototype that would be good.